docs/moisture_dataset.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+""" Script to merge AR ML datasets into CO ML dataset
```

Comment:
Minor nit: There should be no space after the `"""`.

Reply:
Fixed.